libtelemetry/src/main/java/com/mapbox/android/telemetry/CertificatePinnerFactory.java

```diff
@@ -61,13 +61,11 @@ private void addCertificatesPins(Map<String, List<String>> pins, CertificatePinn
     CertificateBlacklist certificateBlacklist = new CertificateBlacklist(MapboxTelemetry.applicationContext,
       MapboxTelemetry.accessToken);
 
-    ArrayList blackList = certificateBlacklist.retrieveBlackList();
+    List blackList = certificateBlacklist.retrieveBlackList();
 
-    if (blackList != null) {
```

Comment:
Shouldn't we still check if the `blackList` is empty? We're traversing `hashList` even if `blackList` has no elements which I guess is not needed. Does that make sense?